app/updatemanager.go

```diff
@@ -132,6 +135,9 @@ func (u *UpdateManager) run(ctx context.Context) error {
 				UpdateManagerMap.Set(&controlMap)
 			}
 
+			log.Debugf("Setting the control map parameter to: %s", controlMap.ID)
+			expiry := time.Now().Add(time.Duration(u.updateControlTimeoutSeconds) * time.Second)
+			UpdateManagerMap.SetAndStamp(&controlMap, expiry)
```

Comment:
This does not seem correct to me, we should `Set` in once. Is it WIP or am I miss-understanding?

Reply:
> `we should Set in once`
I'm sorry, I don't fully understand what you mean here? :thinking:

Reply:
That we use `Set` in line 135, and then `SetAndStamp` on line 140. This is probably the result of a wrong conflict resolution when pulling in my changes...

Reply:
Oh, I see :)

No it is not:
* The time is updated to the current time each time the map is updated by an outside source (not by Mender itself).
So only maps from the DBus inlet should be stamped